### ISSUE TITLE
Proposal: btop

### ISSUE THREAD

**Issue by FlyingHavoc** [ibraheemdev/modern-unix#69]
Resource monitor that shows usage and stats for processor, memory, disks, network and processes.
C++ version and continuation of bashtop and bpytop.
https://github.com/aristocratos/btop

**Comment by Thelvaen** [comment]
I'm voting on that one too, it's a native binary, and does not have external dependencies on npm/javascript interpreter.

**Comment by cosmojg** [comment]
Yeah, why isn't this here? Definitely the best top-like experience there is!
